engine/apps/api/label_filtering.py

```diff
@@ -0,0 +1,13 @@
+def parse_label_query(labelQuery):
+    """
+    parse_label_query returns list of key-value tuples from a request 'label' query param.
+    It expects label query param to be an array of strings – key-value pairs separated with ':'.
```

Comment:
This function doesn't have to be aware of the query param name `'label'`, right?

`It expects label query param to be an array of strings` -> maybe it's easier to add some type annotations instead?

Reply:
It's not about name of the query param, but about some query param which carries label query